deploy/helm/grafana-operator/templates/cm.yaml

```diff
@@ -5,9 +5,8 @@ metadata:
   name: {{ include "grafana-operator.fullname" . }}
   namespace: {{ include "grafana-operator.namespace" . }}
   labels:
-    {{- with .Values.additionalLabels }}
-    {{- toYaml . | nindent 4 }}
-    {{- end }}
+    {{- include "grafana-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: config
```

Comment:
I don't agree with your interpretation of how `app.kubernetes.io/component` should be used.
The [component](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels) is related to the whole object, in this case `grafana-operator`. It doesn't differ between object like rbac or network.

Reply:
@NissesSenap so what do you like for this label ? just `grafana-operator` or `operator` ?

Reply:
`operator` I guess

Reply:
done.